R/msnset2se.R

```diff
@@ -5,22 +5,23 @@
 #' @param se SummarizedExperiment,
 #' Object which will be turned into an MSnSet object.
 #' @return MSnSet object.
-#' @export
 se2msnset <- function(se) {
   # Check input
-  assertthat::assert_that(inherits(se, "SummarizedExperiment"))
+  stopifnot(inherits(se, "SummarizedExperiment"))
+	
+	if(!requireNamespace("SummarizedExperiment")) {
+		stop("The SummarizedExperiment package is required for SummarizedExperiment to MSnSet conversion.")
```

Comment:
Sorry for being petty but could you please use just 4 spaces for indentation (you used ~8) and keep lines shorter than 80 characters?

Reply:
BTW: if you use `setAs` instead of an internal function the `stopifnot(inherits(se, "SummarizedExperiment"))` is superfluous.